src/components/NewLocationPage/CountyMap/CountyMap.tsx

```diff
@@ -61,6 +61,24 @@ const MAP_TYPE_INFO: { [key in MapType]: MapTypeInfo } = {
   },
 };
 
+function useMapTileOffset(): number {
+  const [yPosition, setYposition] = useState(0);
+  const pageHeight = document.documentElement.scrollHeight;
+  const footerHeight = document.getElementsByTagName('footer')[0]?.scrollHeight;
+  // Add an additional 500px to detect when the user is close enough to the bottom of the page.
```

Comment:
This comment is hard to understand without more context about what this code is doing.  It might be worth combining the comment at line 70 to this one and saying something like:

> Once the user scrolls far enough that the footer is within 500px of the top of the screen, we want to freeze the map tile so that it can't overlap with the footer and obscure footer content. To do that we set its `bottom` to be 100px above the footer.